released/packages/coq-mathcomp-real-closed/coq-mathcomp-real-closed.1.0.3/opam

```diff
@@ -0,0 +1,31 @@
+opam-version: "2.0"
+name: "coq-mathcomp-real-closed"
+maintainer: "Mathematical Components <mathcomp-dev@sympa.inria.fr>"
+
+homepage: "https://github.com/math-comp/real-closed"
+bug-reports: "https://github.com/math-comp/real-closed/issues"
+dev-repo: "git+https://github.com/math-comp/real-closed.git"
+license: "CeCILL-B"
+
+build: [ make "-j" "%{jobs}%" ]
+install: [ make "install" ]
+depends: [
+  "coq-mathcomp-field" { (>= "8.7" & < "8.11~") }
+  "coq-mathcomp-field" { (>= "1.8.0" & < "1.10.0~") }
+  "coq-mathcomp-bigenough"   {(>= "1.0.0" & < "1.1.0~")}
+]
+
+tags: [ "keyword:real closed field" "keyword:small scale reflection" "keyword:mathematical components" "date:2019-05-23" "logpath:mathcomp"]
```

Comment:
```suggestion
tags: [ "keyword:real closed field" "keyword:small scale reflection" "keyword:mathematical components" "date:2019-05-23" "logpath:mathcomp.real_closed"]
```
For some more specificity (useful when suggesting packages to install based on failing `Require`).